pkg/sfu/sequencer_test.go

```diff
@@ -15,33 +15,39 @@ func Test_sequencer(t *testing.T) {
 	seq := newSequencer(500, logger.Logger(logger.GetLogger()))
 	off := uint16(15)
 
-	for i := uint16(1); i < 520; i++ {
-		seq.push(i, i+off, 123, 2, true)
+	for i := uint16(1); i < 518; i++ {
+		seq.push(i, i+off, 123, 2)
 	}
+	// send the last two out-of-order
```

Comment:
Modifying this a bit and also adding a case below to force out-of-order packets into the sequencer and make sure that it works.